src/Providers/EntityManagerProvider.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Choredo\Providers;
+
+use Doctrine\Common;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\ORM;
+use Illuminate\Support\ServiceProvider;
+use Ramsey\Uuid\Doctrine\UuidType;
+
+/**
+ * Class EntityManagerProvider
+ * @package Choredo\Providers
+ */
+class EntityManagerProvider extends ServiceProvider
+{
+    public function register()
+    {
+        $environment = getenv('CHOREDO_ENVIRONMENT') ?? 'production';
+        $isProduction = $environment === 'production';
+
+        $entityDirectory = realpath(__DIR__ . '/../Entities');
+        $proxyDirectory = realpath(__DIR__ . '/../proxies/');
+
+        Type::addType('uuid', UuidType::class);
+        $cache = new Common\Cache\ArrayCache();
+
+        $config = new ORM\Configuration();
+        $annotationDriver = $config->newDefaultAnnotationDriver($entityDirectory, false);
+        $config->setMetadataDriverImpl($annotationDriver);
+        $config->setQueryCacheImpl($cache);
+        $config->setMetadataCacheImpl($cache);
+        $config->setProxyDir($proxyDirectory);
+        $config->setProxyNamespace('HelpMeAbstract\Proxies');
+
+        if ($isProduction) {
+            $config->setAutoGenerateProxyClasses(Common\Proxy\AbstractProxyFactory::AUTOGENERATE_NEVER);
+        } else {
+            $config->setAutoGenerateProxyClasses(Common\Proxy\AbstractProxyFactory::AUTOGENERATE_ALWAYS);
+        }
+
+        $dbParams = [
+            'driver'   => 'pdo_pgsql',
+            'user'     => getenv('CHOREDO_DB_USERNAME'),
```

Comment:
I believe the Lumen setup has these defaulting to DB_XXXXX, which is what I used in the Docker setup. We need to change it either here or there. 

https://github.com/greydnls/choredo-api/blob/4a3984c251b0801362e479006d83c6eb8b2e00ef/docker-compose.yml#L6-L8